src/components/Sections/Home/DonationPopup.js

```diff
@@ -0,0 +1,177 @@
+          <p>Por eso ahora, m&aacute;s que nunca, les invitamos a apoyar econ&oacute;micamente, participando as&iacute; en la expansi&oacute;n de la difusi&oacute;n de nuestros contenidos a la humanidad.</p>
+          <p>De esta manera, todos y cada uno de nosotros podemos cumplir el sue&ntilde;o de los cabalistas de difundir la sabidur&iacute;a de la Cabal&aacute; a las masas y esparcir una luz de seguridad en el mundo, tal como lo escribi&oacute; el cabalista Rab&iacute; Yehuda Ashlag (Baal HaSulam): &quot;S&oacute;lo a trav&eacute;s de la expansi&oacute;n de la sabidur&iacute;a de la Cabal&aacute; a las masas obtendremos la completa redenci&oacute;n&rdquo;.&nbsp;&nbsp;</p>
+          <p>En esta ocasi&oacute;n agradecemos a todos los amigos y amigas que alguna vez contribuyeron o que lo siguen haciendo regularmente, y agradecemos de antemano a todos los nuevos colaboradores por su importante aporte a la difusi&oacute;n de la sabidur&iacute;a de la Cabal&aacute;.</p>
+          <p><b>La Direcci&oacute;n de la Asociaci&oacute;n Bnei Baruj</b></p>
+        </div>;
+      default:
+        return <div>
+          <p>Dear Friends,</p>
+          <p>Before you scroll further and enjoy the original contents on the site, we would like to invite you to
+            become
+            our partners in the dissemination enterprise of the "Bnei Baruch - Kabbalah La'am" association.</p>
+          <p>To date we have distributed unprecedented amounts of valuable and unique material all over the world, and
+            it is
+            all thanks to you, thousands of loyal volunteers.</p>
+          <p>But it is not enough. Day by day the world falls into a more acute crisis in all areas of life, and it
+            needs
+            content that only our organization can provide; content that explains the importance of the connection
+            between
+            people and brings them closer to solving all crises.</p>
+          <p>That is why now, more than ever, we invite you to donate financially and participate in increasing the
+            dissemination of our content to humanity.</p>
+          <p>In this way, each and every one of us can fulfill the Kabbalists' dream to spread the wisdom of Kabbalah to
+            the
+            masses and pour the light of confidence to the world, as the Baal HaSulam wrote: "Only by the expansion of
+            the
+            wisdom of Kabbalah will we be rewarded with complete redemption."</p>
+          <p>On this occasion we wish to thank the friends who have donated and keep donating on a regular basis, and we
+            thank in advance everyone who now joins in for your important contribution to the dissemination of the
+            wisdom of
+            Kabbalah.</p>
+          <p><b>The management of the "Bnei Baruch - Kabbalah La'am" association.</b></p>
+        </div>;
+    }
+  }
+
+  return (
+    <Modal
+      closeIcon
+      className={clsx('donationPopup', {
```

Comment:
if you will use attribute dir instead className its not work?

Reply:
the clasName is for style + mobile device